commands/initial/templates/examples/example.component.html

```diff
@@ -0,0 +1 @@
+<div class="welcome">Welcome</div>
```

Comment:
The intent here was that we have a default example, much like the angular-cli. That was also the intent of the default test above. I can remove if not needed